mungegithub/mungers/release-note-label_test.go

```diff
@@ -244,17 +244,22 @@ func TestGetReleaseNote(t *testing.T) {
 		expectedReleaseNoteVariable string
 	}{
 		{
-			body:                        "**Release note**: <other unimportant information> ```NONE```",
+			body:                        "**Release note**:  ```NONE```",
 			expectedReleaseNote:         "NONE",
 			expectedReleaseNoteVariable: releaseNoteNone,
 		},
 		{
-			body:                        "**Release note**: <other unimportant information> ```This is a description of my feature```",
+			body:                        "**Release note**:\n\n ```NONE```",
+			expectedReleaseNote:         "NONE",
+			expectedReleaseNoteVariable: releaseNoteNone,
+		},
+		{
+			body:                        "**Release note**:\n\n  ```This is a description of my feature```",
 			expectedReleaseNote:         "This is a description of my feature",
 			expectedReleaseNoteVariable: releaseNote,
 		},
 		{
-			body:                        "**Release note**: <other unimportant information> ```This is my feature. There is some action required for my feature.```",
+			body:                        "**Release note**: ```This is my feature. There is some action required for my feature.```",
```

Comment:
can we add a test that leads to no release note? One that shows that the regex is less lenient?